Cargo.lock

```diff
@@ -1399,6 +1425,15 @@ dependencies = [
  "void",
 ]
 
+[[package]]
+name = "nom"
+version = "3.2.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "05aec50c70fd288702bcd93284a8444607f3292dbdf2a30de5ea5dcdbe72287b"
+dependencies = [
+ "memchr 1.0.2",
+]
```

Comment:
I didn't realize that systemstat depends on nom, and a very old version of nom at that.

Looking through their source code they have some very ridiculous macros
https://github.com/myfreeweb/systemstat/blob/a7c54aa0d052989e93f1ef84eb1686fbbfeb92f9/src/platform/linux.rs#L62-L82

This does not give me confidence...